vizro-ai/src/vizro_ai/plot/_response_models.py

```diff
@@ -93,11 +93,18 @@ class ChartPlan(BaseModel):
 
     @validator("chart_code")
     def _check_chart_code(cls, v):
+        # Remove markdown code block if present
```

Comment:
ChatGPT just suggested me this:

```
import re

def strip_markdown(v):
    # Strip leading and trailing markdown code block delimiters if they exist
    return re.sub(r"^```(?:python\n)?|```$", "", v).strip()
```

or 

```
def strip_markdown(v):
    if v.startswith("```python\n"):
        v = v[len("```python\n"):]
    elif v.startswith("```\n"):
        v = v[len("```\n"):]
    if v.endswith("```"):
        v = v[:-3]
    return v.strip()
```

Maybe better?